packages/fast-components-react-base/src/image/README.MD

```diff
@@ -1,31 +1,8 @@
-# Implementation
+## Best practices
+Use *image* to break up text-heavy pages, to illustrate products for sale, or to improve storytelling on a page. The use of *image* should always support the page’s story and, when possible, contribute to that story. Size the *image* to complement the rest of the page content. Use the **picture** option to employ a selection of image versions for different viewport sizes.
 
-## Implementation example
-```javascript
+### Style guidance
+Don't use *image* as a substitute for *hero*, *feature*, or *product* and *content placements*. Don't use an image if it reduces the effectiveness of the page.

```

Comment:
FYI - created an issue #473  to scrub documentation for non-existent components. We'll handle this in a different PR.